pkg/sessionctx/variable/sysvar.go

```diff
@@ -3136,11 +3136,10 @@ var defaultSysVars = []*SysVar{
 	},
 	{Scope: ScopeInstance, Name: TiDBServiceScope, Value: "", Type: TypeStr,
 		Validation: func(_ *SessionVars, normalizedValue string, originalValue string, _ ScopeFlag) (string, error) {
-			_, ok := distroleutil.ToTiDBServiceScope(originalValue)
-			if !ok {
-				err := fmt.Errorf("incorrect value: `%s`. %s options: %s",
+			if !servicescope.IsValidServiceScope(originalValue) {
+				err := fmt.Errorf("incorrect value: `%s`. %s. It must be 64 characters or fewer and consist only of letters (a-z, A-Z), numbers (0-9), hyphens (-), and underscores (_)",
```

Comment:
change to `CheckServiceScope` which returns error if not valid? so we can reuse the message as in main.go